src/styles/_font-faces.scss

```diff
@@ -1,20 +1,4 @@
 @font-face {
-  font-family: "Museo";
-  font-style: normal;
-  font-weight: 300;
-  src: url("https://images-ra.adoptapet.com/fonts/museo-regular-300.woff2") format("woff2");
-}
-
-@font-face {
-  font-family: "Museo";
-  font-style: normal;
-  font-weight: 500;
-  src: url("https://images-ra.adoptapet.com/fonts/museo-regular-500.woff2") format("woff2");
-}
-
-@font-face {
-  font-family: "Museo";
-  font-style: normal;
-  font-weight: 700;
-  src: url("https://images-ra.adoptapet.com/fonts/museo-regular-700.woff2") format("woff2");
+  font-family: "special";
```

Comment:
What is this?  I am confused about what "special" is?  I also don't see the new font faces: 
- Soleil
- Bayu Prahara

Reply:
(Answered in my first comment.)

Reply:
Shouldn't we still be adding the new font faces here because they are used by `$font` and `$font-special` or are those being defined somewhere else that I am missing.